src/core/lib/attribute-manager.js

```diff
@@ -311,7 +311,13 @@ export default class AttributeManager {
    * @return {Object} attributes - descriptors
    */
   getAttributes() {
-    return this.attributes;
+    const attributes = {};
```

Comment:
Do we want two accessors `getAttributeArrays` and `getAttributeBuffers`? Or at least rename the one we have?

Reply:
Also `attributrTransitionManager.getAttributes` returns buffers

Reply:
This is more in the context of long term discussion. Perhaps some context can help:

My long-term view is that the AttributeManager should not rely on luma.gl to handle buffer creation, and instead provide a function that only returns buffers. 

Apart from simply wanting to keep this complex class easier to maintain by reducing the number of data variants it handles there are two important use cases:
* For really big data, we could reuse a single typed array (or `ArrayBuffer`) instead of allocating and holding on to each array and Buffer combination. This "scratch" array could then be reused by the next layer etc. Almost 2x memory reduction.
* For the partial update use case, we need to call `buffer.subData` to efficiently update the WebGL buffers. This is only possible if `AttributeManager` owns the buffers.

> Also attributeTransitionManager.getAttributes returns buffers

Sounds like making AttributeManager also return only buffers would be a nice alignment then.